### PR TITLE
fix(files): Use OCP\Util::getForbiddenFileNameChars instead of directaccess to system config

### PR DESCRIPTION
**Pull request by susnux**
## Summary

We (might) extend it in the future so we should only use one common place (OCP\Util) for getting forbidden characters.

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [x] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [x] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

File: apps/files/tests/Controller/ViewControllerTest.php
Comment (artonge) [inline apps/files/tests/Controller/ViewControllerTest.php:1]: Why are those changes needed?

**Review by artonge — APPROVED** [review]

**Review by Altahrim — APPROVED** [review]

File: apps/files/tests/Controller/ViewControllerTest.php
Comment (susnux) [reply apps/files/tests/Controller/ViewControllerTest.php:1]: I thought the test would (like usual) also check initial state and thus we need to ensure a defined value is set. But the state is not checked so no need here.
Removed that change.

**Review by come-nc — APPROVED** [review]
